test/bundler/bundler_footer.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe } from "bun:test";
+import { itBundled } from "./expectBundled";
+
+describe("bundler", () => {
+  itBundled("footer/CommentFooter", {
+    footer: "// developed with love in SF",
+    files: {
+      "/a.js": `console.log("Hello, world!")`,
+    },
+    onAfterBundle(api) {
+      api.expectFile("out.js").toContain("// developed with love in SF");
+    },
+  });
+  itBundled("footer/MultilineFooter", {
+    footer: `/**
+ * This is copyright of [...] ${new Date().getFullYear()}
+ * do not redistribute without consent of [...]
+*/`,
+    files: {
+      "index.js": `console.log("Hello, world!")`,
+    },
+    onAfterBundle(api) {
+      api.expectFile("out.js").toContain(`/**
```

Comment:
should it be toEndWith()?

Reply:
err yes technically for the test, as a note the footer is not always at the end, if you add sourcemaps those go under the footer but not using it in the test case so will change thanks for noticing this